src/components/Controls/ArrayInput/index.tsx

```diff
@@ -0,0 +1,56 @@
+import {FunctionComponent} from "react"
+import CloseIcon from "../../../assets/icons/CloseIcon"
+import Input from "../Input"
+import Button from "../Button"
+import useArrayInput from "./hooks"
+import "./styles.scss"
+
+export type ArrayInputChangeListener = (newValue: ArrayInputValue) => void
```

Comment:
Again, why do you need to define these types?

Reply:
We keep all the types in `types` folder. However, these ones are too simple to define.